lib/interpreters/split_clustering.rb

```diff
@@ -288,7 +283,8 @@ def self.split_solve_sub_vrp(service_vrp, job = nil, &block)
 
       service_vrp[:vrp] = create_sub_vrp(ss_data)
 
-      OptimizerWrapper.define_process(service_vrp, job, &block)
+      solution = OptimizerWrapper.define_process(service_vrp, job, &block)
+      solution.parse(service_vrp[:vrp])
```

Comment:
why do the solution need the vrp to be able to parse itself? Since a solution belongs to a specific VRP, I think we can have the "reference" of the VRP inside the solution object so that, we don't need to pass the vrp to it for different operations, what do you think?

Reply:
See internal ticket 899